articles/libraries/lock-android/v1/internationalization.md

```diff
@@ -0,0 +1,37 @@
+---
+section: libraries
+toc_title: Internationalization
+title: Internationalization
+description: Internationalization support in Lock for Android
+---
+# Internationalization in Lock for Android v1
```

Comment:
Here the titles are aligned but the version is not needed. 

Also, at the previous title was `Lock v1 Configuration` while this is `Internationalization in Lock for Android v1` (they are both for Android though).

Pick one format and use it in all titles.